conn_test.go

```diff
@@ -145,6 +145,9 @@ func newTestSession(addr string, proto protoVersion) (*Session, error) {
 }
 
 func TestDNSLookupConnected(t *testing.T) {
+	if *flagSkipLong {
+		t.Skip("Skipping long test")
+	}
```

Comment:
Can we just reduce some timeouts for this test? How long does it take to complete?

Reply:
Now using testing.Short() to test for the option.